models/settings.luis.json

```diff
@@ -1,11 +1,14 @@
 {
-  "prebuiltEntityTypes": {
-    "number": "number"
+  "roles": {
+    "songCount": "number"
   },
   "appTemplate": {
     "prebuiltEntities": [
       {
-        "name": "number"
+        "name": "number",
+        "roles": [
+          "songCount"
+        ]
```

Comment:
Do we have to add documentation for this change?

Reply:
Yes.

Reply:
Done.